--- util.h.old	2014-08-10 23:30:39.041435000 -0400
+++ util.h	2014-08-10 23:35:54.000000000 -0400
@@ -34,6 +34,7 @@
 #include <limits.h>
 #include <sys/stat.h>
 #include <dirent.h>
+#include <pthread_np.h>

 #include "macro.h"

@@ -375,7 +376,7 @@

 int pipe_eof(int fd);

-cpu_set_t* cpu_set_malloc(unsigned *ncpus);
+cpuset_t* cpu_set_malloc(unsigned *ncpus);

 void status_vprintf(const char *status, bool ellipse, const char *format, va_list ap);
 void status_printf(const char *status, bool ellipse, const char *format, ...);
